e2e/nomostest/reset.go

```diff
@@ -441,6 +454,19 @@ func withLabelListOption(key, value string) client.MatchingLabelsSelector {
 	return client.MatchingLabelsSelector{Selector: labelSelector}
 }
 
+func withNameNotInListOption(values ...string) client.MatchingFieldsSelector {
```

Comment:
Looks like the input arg is unused here and it's always using `sharedTestNamespaces`. Should update this to use the input arg

Reply:
good catch

Reply:
Fixed